policies/export_eventhub_defender_logs/README.md

```diff
@@ -0,0 +1,59 @@
+# HMCTS Deploy export to eventHub for Defender for Cloud data
+
+Creates a Microsoft.Security/Automation and Resource Group that will forward Defender for Cloud data to a specified event hub.
+
+An existing resource can be used but must exist in every subscription that the policy is applied to. If using this method, make sure to set `Create resource group` to `false`.
+
+## Required Parameters
+
+```yaml
+- Name: Resource group name
+  Type: String
+
+- Name: Event Hub details
+  Type: String
+```
```

Comment:
Worth mentioning that `Event Hub details` is just resource ID
Was going to suggest a var naming change but can imagine that's a pain - maybe extra description here will do it?